src/playlist-manager-qt/playlist-manager-qt.cc

```diff
@@ -324,7 +323,7 @@ static QToolButton * new_tool_button (const char * text, const char * icon)
     auto button = new QToolButton;
     button->setIcon (audqt::get_icon (icon));
     button->setText (audqt::translate_str (text));
-    button->setToolButtonStyle (Qt::ToolButtonTextBesideIcon);
+    button->setToolButtonStyle (Qt::ToolButtonFollowStyle);
```

Comment:
Only this isn't really a toolbar.  We don't want to follow the toolbar style here, we want to follow the button style.

Reply:
Where do you get the toolbar reference from? All this change does is to instruct the new tool*button* to follow the toolbutton style so that users aren't obliged to waste horizontal space on text labels if they don't want that.